MedicalDocManagment.WebUI/app/models/position.model.ts

```diff
@@ -2,7 +2,7 @@
   id: string;
   name: string;
 
-  constructor(jsonobject) {
+  constructor(jsonobject=null) {
```

Comment:
Good, thanks.
But also exists one more way to do it. It's like so `constructor(jsonobject?) `. It just looks better.